packages/elpi/elpi.1.4.1/opam

```diff
@@ -0,0 +1,82 @@
+opam-version: "2.0"
+name: "elpi"
```

Comment:
```suggestion
```

This field is redundant. Opam will override it with the name inferred from the path

Reply:
done